homeassistant/components/deconz/__init__.py

```diff
@@ -186,3 +187,92 @@ def deconz_shutdown(event):
         entity_picture="/static/images/logo_deconz.jpeg",
```

Comment:
Please remove all the `configurator` code.

Reply:
Is it ok that I do this when I've migrated the discovery part?

Reply:
Sure